recipes/opty/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "opty" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "362a65e7b313d3a8fb403ae8148035c40c6078d1ca490b900b78a439110ff16e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.8.1
```

Comment:
I guess I'm confused here. If the setup.py has `install_requires` items in it, don't they have to be installed in the build? Or do they?  If not, I can remove them. Or should the build only have items that are effectively imported in the setup.py file?